scripts/time-filter.js

```diff
@@ -0,0 +1,88 @@
+import http from 'k6/http';
+import {
+    check,
+    sleep
+} from 'k6';
+import {
+    generateRandomCoordinate,
+    commonOptions,
+    getCapitalCoordinates,
+    destinations,
+    configs,
+    setThresholdsForScenarios,
+    summaryFormatter
+} from './common.js';
+
+export let options = configs
```

Comment:
Can you generate tests with different amount of destinations here?

Reply:
Yes, using `-e DESTINATIONS="100,150,200"` will change the destinations amount, because in common.js -> destinations are taken from an environment variable. Tried and confirmed myself.
![image](https://github.com/traveltime-dev/traveltime-benchmarks/assets/140691866/314003b1-0cb7-474e-ba56-3bef2339671e)